src/other/utils.jl

```diff
@@ -58,3 +58,42 @@ function gennames(n::Integer)
     return res
 end
 
+"""
+    flatten(df::AbstractDataFrame, veccol::Union{Integer, Symbol})
+
+When column `veccol` of `df` has elements non-zero length, for example a `Vector` 
```

Comment:
```suggestion
When column `veccol` of `df` has iterable elements that define `length`, for example a `Vector` 
```
I understand we allow zero-length iterables and we only require that they support `length` (as in general some iterables do not support `length`)